autoload/neomake/makers/ft/haskell.vim

```diff
@@ -109,6 +109,7 @@ function! neomake#makers#ft#haskell#HlintEntryProcess(entry) abort
     " Postprocess hlint output to make it more readable as a single line
     let a:entry.text = substitute(a:entry.text, '\v(Found:)\s*\n', ' | \1', 'g')
     let a:entry.text = substitute(a:entry.text, '\v(Why not:)\s*\n', ' | \1', 'g')
+    let a:entry.text = substitute(a:entry.text, 'No hints', '', 'g')
```

Comment:
I think the regex pattern should be more strict, e.g. by matching the beginning/end of line etc.

/ cc @dkasak